checkpoint/dev.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install -U flask-cors
```

Comment:
Two questions on this. Should it be in -requirements-dev instead of here? / should this script install all of requirements dev given it is the 'dev' script.

Is installing this all that is needed to use it? I don't see it used anywhere explicitly

Reply:
I think `requirements-dev` is really installing deps for CI, so this doesnt make sense there. These are not actually needed here to run in dev mode. 

This + running `pip install -e .` is all that is required (this is noted in the FE README).